### PR TITLE
Jinja2 Template, IPv6 pull-filters, skip ipv6 leak protection when ipv6 disabled

### LINKED ISSUE #162: [BUG] Fix connection on IPv6 disabled systems
Add

```
pull-filter ignore "ifconfig-ipv6 " 
pull-filter ignore "route-ipv6 "
```

to `template.ovpn` if `net.ipv6.conf.all.disable_ipv6=1`

### PR DESCRIPTION
**Pull request by Rafficer**
This PR does multiple things, as one change is needed for the other.

- Use a Jinja2 template for OpenVPN configuration instead of modifying the config the API provides
- Add IPv6 pull filters to the config when IPv6 is disabled or unavailable system wide
- Don't activate IPv6 leak protection when IPv6 is disabled or unavailable system wide

### CODE REVIEW

File: protonvpn_cli/utils.py [hunk 1]
```diff
@@ -302,6 +302,16 @@ def make_ovpn_template():
 
     logger.debug("remote and proto lines removed")
 
+    # Issue #162
+    if os.path.isfile("/proc/sys/net/ipv6/conf/all/disable_ipv6"):
+        ipv6_state = subprocess.run(
+            ["sysctl", "-n", "net.ipv6.conf.all.disable_ipv6"],
+            stdout=subprocess.PIPE)
+        if int(ipv6_state.stdout.decode("utf-8")):
```
Comment (kaplun) [inline protonvpn_cli/utils.py:310]: Can you log that you found out that IPv6 is disabled?

**Comment by 8176135** [comment]
This fix I believe won't work if ipv6 is disabled with kernel boot parameters? 

As the `/proc/sys/net/ipv6/` folder won't exist.

**Comment by Rafficer** [comment]
Yes, that is why I'll rework it later :P

But thanks for noticing as well! :)

File: protonvpn_cli/utils.py [hunk 2]
```diff
@@ -302,6 +302,17 @@ def make_ovpn_template():
 
     logger.debug("remote and proto lines removed")
 
+    # Issue #162
+    if os.path.isfile("/proc/sys/net/ipv6/conf/all/disable_ipv6"):
```
Comment (kaplun) [inline protonvpn_cli/utils.py:306]: If the file does not exists, doesn't it mean that IPv6 is not supported by the system and hence also in this case we need to add the pull-filters ?

Comment (Rafficer) [reply protonvpn_cli/utils.py:306]: True as well. I'll rework this tomorrow with discussed changes.

**Comment by kaplun** [comment]
@Rafficer we plan to add the 2 pull filters directly in the configuration that is provided via the API, because otherwise routers with IPv6 disabled currently won't work with IPv6 servers.

So if you actually as well you are fetching the configuration from the API then you need to make this fix a bit smarter. 

Actually in general I wouldn't use the configuration from the API but would keep an independent template in linux-cli (if this isn't already the case), because you have much more control of the Linux installation (the API can only provide a generic configuration that should fit without being touched, the majority of setups.

**Comment by kaplun** [comment]
FYI in the next days the new configuration will contain:
```
## BEGIN-IPV6: Comment out the following two lines if you want to exploit IPv6 support provided by an IPv6-aware server
pull-filter ignore "ifconfig-ipv6"
pull-filter ignore "route-ipv6"
## END-IPV6
```
So this functionality should rather be reversed...

**Comment by Rafficer** [comment]
I'm working on providing a custom config via jinja2, so as long as I can get there before that change is live, we're fine.

**Pull request merged**